specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml

```diff
@@ -0,0 +1,12 @@
+emit:
+  - '@azure-tools/typespec-autorest'
+options:
+  '@azure-tools/typespec-autorest':
+    use-read-only-status-schema: true
+    emitter-output-dir: "{project-root}/.."
```

Comment:
Are all these properties necessary even in this basic case? Or can some of them rely on defaults?

Reply:
`use-read-only-status-schema: true`  -> Mitigates the ProvisioningStateReadOnly lint error
`emitter-output-dir: "{project-root}/.."` and  ` output-file: `  -> Need to for correct swagger file placement